src/websocket/server.cc

```diff
@@ -43,6 +43,55 @@ static sstring http_upgrade_reply_template =
 
 static logger wlogger("websocket");
 
+uint8_t opcode_to_uint8(opcodes opcode) {
+    /*
+     * Opcodes defined here:
+     * https://datatracker.ietf.org/doc/html/rfc6455#section-5.2
+     */
+    switch (opcode) {
+        case opcodes::CONTINUATION:
+            return 0x0;
```

Comment:
I don't think you need this function. enum class can have a deterministic mapping (the first one is 0 which you can assign explicitly, the next one is 1, and so on) so you don't need such a switch(). At least I think...

Reply:
@astalke in other words, move the mappings straight to the enum definition in server.hh and drop this function:
```cpp
enum class opcodes {
    CONTINUATION = 0x0,
    TEXT = 0x1,
    BINARY = 0x2,
    CLOSE = 0x8,
    (...and so on)
};

```